spec/factories/season.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :season do
-    name { Date.today.year.to_s }
+    sequence(:name) { |n| "#{Date.today.year}-#{n}" }
```

Comment:
I think the name has to correspond to a year for its automatic lookup to work, no?

Reply:
Yes. I even first thought about excluding the `season` model from the factory linting. But then I thought, there are some explicit tests for this functionality in place anyways and the DB is wiped after each test anyways, so I thought it would be fine.

Can undo it if you have concerns about it ✌️

Reply:
You're right about the flushing after each example. I'm not sure if anyone looks into the factory definitions to understand the inner workings of a class, but it would confuse me. I guess I'd prefer sticking with `%r(\d{4})` but we should also add a format validation (not necessarily in this PR)